examples/tts/conf/fastpitch_align.yaml

```diff
@@ -0,0 +1,183 @@
+  max_token_duration: 75
+  n_mel_channels: 80
+  pitch_embedding_kernel_size: 3
+  n_window_size: 2048
+  n_window_stride: 512
+
+  train_ds:
+    dataset:
+      _target_: "nemo.collections.asr.data.audio_to_text.AudioToCharWithPriorAndPitchDataset"
+      manifest_filepath: ${train_dataset}
+      max_duration: null
+      min_duration: 0.1
+      int_values: false
+      normalize: true
+      sample_rate: ${sample_rate}
+      trim: false
+      sup_data_path: ${prior_folder}
+      n_window_stride: ${model.n_window_stride}
+      n_window_size: ${model.n_window_size}
+      pitch_fmin: 80
+      pitch_fmax: 640
+      pitch_avg: 216.23100173462166
+      pitch_std: 46.24582950436719
+      vocab:
+        notation: phonemes
+        punct: true
+        spaces: true
+        stresses: true
+        add_blank_at: None
+        pad_with_space: True
+        chars: true
+        improved_version_g2p: true
+    dataloader_params:
+      drop_last: false
+      shuffle: true
+      batch_size: 32
+      num_workers: 12
+
+
+  validation_ds:
+    dataset:
+      _target_: "nemo.collections.asr.data.audio_to_text.AudioToCharWithPriorAndPitchDataset"
+      manifest_filepath: ${validation_datasets}
+      max_duration: null
+      min_duration: null
+      int_values: false
+      normalize: true
+      sample_rate: ${sample_rate}
+      trim: false
+      sup_data_path: ${prior_folder}
+      n_window_stride: ${model.n_window_stride}
+      n_window_size: ${model.n_window_size}
+      pitch_fmin: 80
+      pitch_fmax: 640
+      pitch_avg: 216.23100173462166
+      pitch_std: 46.24582950436719
+      vocab:
+        notation: phonemes
+        punct: true
+        spaces: true
+        stresses: true
+        add_blank_at: None
+        pad_with_space: True
+        chars: true
+        improved_version_g2p: true
+    dataloader_params:
+      drop_last: false
+      shuffle: false
+      batch_size: 32
+      num_workers: 8
+
+  preprocessor:
+    _target_: nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor
+    dither: 0.0
+    features: ${model.n_mel_channels}
+    frame_splicing: 1
+    highfreq: null
+    log: true
+    log_zero_guard_type: add
+    log_zero_guard_value: 1e-05
+    lowfreq: 0
+    mag_power: 1.0
+    n_fft: ${model.n_window_size}
+    n_window_size: ${model.n_window_size}
+    n_window_stride:  ${model.n_window_stride}
+    normalize: null
+    pad_to: 1
+    pad_value: 0
+    preemph: null
+    sample_rate: ${sample_rate}
+    window: hann
+    window_size: null
+    window_stride: null
+
+  input_fft:
+    _target_: nemo.collections.tts.modules.transformer.FFTransformerEncoder
+    n_layer: 6
+    n_head: 1
+    d_model: ${model.symbols_embedding_dim}
+    d_head: 64
+    d_inner: 1536
+    kernel_size: 3
+    dropout: 0.1
+    dropatt: 0.1
+    dropemb: 0.0
+    # n_embed: 148  # NOTE Should match # of tokens in `symbol_set`
+    d_embed: ${model.symbols_embedding_dim}
+    # padding_idx: 0
```

Comment:
Let's remove commented lines and leave a comment that `padding_idx` and `n_embed` will be set in `__init__` of model

Reply:
done